### DISCUSSION TITLE
Why SPEI?

### DISCUSSION THREAD

**Discussion by HaireLab** [HaireLab/community-stress-gradients#4] [General]
Starting a discussion of climate indices here--what will work best for our purpose? We are focused on indices that represent water balance. So far, just some background.

SPEI: Difference between precipitation and potential evapotranspiration for each month. Considered to be a good measure of drought severity in terms of intensity and duration and can identify the onset and end of drought episodes. The SPEI allows comparison of drought severity through time and space, since it can be calculated over a wide range of climates. More information and data found here: See: https://spei.csic.es/database.html and https://climatedataguide.ucar.edu/climate-data/standardized-precipitation-evapotranspiration-index-spei

- Littlefield, Caitlin E. 2019. “Topography and Post-Fire Climatic Conditions Shape Spatio-Temporal Patterns of Conifer Establishment and Growth.” Fire Ecology, 15:34. https://doi.org/10.1186/s42408-019-0047-7

Two other indices of water balance. Both found here: http://thredds.northwestknowledge.net:8080/thredds/catalog/TERRACLIMATE_ALL/data/catalog.html
AET: Reflects the simultaneous availability of biologically usable energy and water. Index of site potential for productivity.
 
DEF: Evaporative demand that is not met by available water and is thus an index of potential effects of drought stress on plants.

- Crimmins, Shawn M, Solomon Z Dobrowski, Jonathan A Greenberg, John T Abatzoglou, and Alison R Mynsberge. 2011. “Changes in Climatic Water Balance Drive Downhill Shifts in Plant Species Optimum Elevations.” Science 331 (6015): 324–27. https://doi.org/10.1126/science.1199040
- Stephenson, N. (1998), Actual evapotranspiration and deficit: biologically meaningful correlates of vegetation distribution across spatial scales. Journal of Biogeography, 25: 855-870. doi:10.1046/j.1365-2699.1998.00233.x

**Comment by megkrawchuk** [comment]
Finally getting myself all modern with github. Yay! 
Helpful to have this summary, Sandra. Looking forward to seeing your thinking on the CWD/DEF ideas at finer spatial resolution from the Lutz method.
This is going to be fun/tricky to quantify the accumulated or maximum/pinch of CWD over time!

**Comment by HaireLab** [comment]
Short follow-up on SPEI--this index seems most suitable for a broad-brush look at trends across years--only one pixel value/month for most study sites/burns. Posted these graphs elsewhere, but will insert one here for reference and a few comments on what we're seeing across sites.

- only two sites had trees recorded for early post-fire years: Roberts Creek and Rodeo
- trees regenerated in locations with variable microclimates (represented by heat load index) across sites
- in some cases, an increase in trees was observed when SPEI indicated a lessening of drought conditions (e.g., Hayman, Missionary Ridge) but greater numbers of trees were observed amidst (possibly milder) drought periods e.g., Roberts Creek 
-
Open for discussion!
![hayman3plots](https://user-images.githubusercontent.com/68163030/122563386-24c7e180-d012-11eb-9138-ed86b1d3d73e.png)

-

**Comment by megkrawchuk** [comment]
Hi Sandra.
Oops, I just saw this now after sending my email response to the group as a whole asking about general patterns that you see in the data!

**Reply by HaireLab** [reply]
no worries--and I liked your idea about looking for interactions btwn spei and hli. 
pondering possible ways to test...someday!
